daemon/src/monitor.rs

```diff
@@ -117,19 +117,29 @@ pub struct Actor {
 struct Cfd {
     id: OrderId,
 
-    monitor_lock_finality: Option<Lock>,
-    monitor_commit_finality: Option<Commit>,
+    lock: Option<Lock>,
+    monitor_lock_finality: bool,
+
+    collaborative_settlement: Option<(Txid, Script)>,
+    monitor_collaborative_settlement_finality: bool,
+
+    commit: Option<Commit>,
+    monitor_commit_finality: bool,
     monitor_cet_timelock: bool,
     monitor_refund_timelock: bool,
-    monitor_refund_finality: Option<Refund>,
+
+    cet: Option<(Txid, Script)>,
+    monitor_cet_finality: bool,
+
+    refund: Option<Refund>,
+    monitor_refund_finality: bool,
+
     monitor_revoked_commit_transactions: Vec<RevokedCommit>,
-    monitor_collaborative_settlement_finality: Option<(Txid, Script)>,
-    monitor_cet_finality: Option<(Txid, Script)>,
 
     // Rebroadcast transactions upon startup
-    lock_tx: Option<Transaction>,
-    cet: Option<Transaction>,
-    commit_tx: Option<Transaction>,
+    broadcast_lock: Option<Transaction>,
+    broadcast_cet: Option<Transaction>,
+    broadcast_commit: Option<Transaction>,
```

Comment:
❓ Why is there no `broadcast_refund`?

Reply:
There was never a `refund: Option<Transaction>` as you can see on the left side of this diff.

I think we want to avoid situations where refund is published automatically after a restart. 

---

Aside from this PR:

To be honest, I'm not sure refund should be published automatically during runtime either. It's rare enough and disrupting enough (doesn't really enforce the bet) that I would argue it should always be triggered manually by either party if the oracle never does its thing. You could theoretically even let the parties settle collaboratively instead.